### PR TITLE
Refactored code

### PR DESCRIPTION
**Pull request by rashidkalwar**
Hi @Davidy22 

I did all the pre-commit stuff looks everything good to me now, please have a thorough look and let me know if I need to make other changes,

hopefully later this week I will on creating functions for that lengthy code in main.py also I will make some classes to make thing a little more simple and accessible.

thanks!

### CODE REVIEW

**Comment by Davidy22** [comment]
Just did a checkout  and the code crashed immediately on my machine when it entered toggleFlag. Will do a quick skim to see what's happening here, but make sure to run the code to make sure there's no semantic breakage.

File: skunkbooth/utils/helpers.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    if isinstance(event, KeyboardEvent):
+        c = event.key_code
+        # Stop on q, esc, ctrl+q and ctrl+w
+        if c in (Screen.KEY_ESCAPE, ord("q"), ctrlQCode, ctrlWCode):
+            raise StopApplication("User pressed quit")
+
+
+def toggleFlag(flag: List[int]) -> None:
+    """Temp function for toggling video recording from inside screen"""
+    vidBuf = Queue(32767)
```
Comment (Davidy22) [inline skunkbooth/utils/helpers.py:26]: vidBuf is a shared pipe between processes that needs to contain messages between processes, in this case a video filename. If you declare a new buffer in here, the value won't get sent out to the shared buffer outside this function. You can add function parameters if they're needed.

**Review by Davidy22 — COMMENTED** [review]
I only found one source of a runtime error but there might be more after you fix it, make sure to run and make sure the program works after you write your patch

File: skunkbooth/utils/helpers.py (hunk 1 above)
Comment (rashidkalwar) [reply skunkbooth/utils/helpers.py:26]: got it. I will do the testing and will let you know.

Comment (Trisanu-007) [reply skunkbooth/utils/helpers.py:26]: What happened with the testing ?

**Comment by Davidy22** [comment]
I'm going to close this because it doesn't look like this is going to get finished. If you have a fixed version of this later you can create a new fresh pull request.

**Pull request closed without merging**